### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MishterKirby**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Not directly (after all, this is **Sprite**Glow), but it's possible if the compo
 
 ![](https://i.gyazo.com/6c92f315d8a25600bf4ec930c5b7de3e.png)
 
-### Why the glow doesn't appear?
+### Why doesn't my sprite glow?
```
Comment (elringus) [inline README.md]: I think the original is more fitting in this case.

File: README.md [hunk 2]
```diff
@@ -40,12 +40,12 @@ Make sure:
 
 ![](https://i.gyazo.com/94fe6e143e310a526b3428c6c62b45bf.png)
 
-### Why the glow appears on the whole image?
+### Why dosen't the glow appears on the whole image?
```
Comment (elringus) [inline README.md]: "dosen't appears" ?

**Review by elringus — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MishterKirby) [reply README.md]: How about "Why doesn't the glow appear?"

File: README.md (hunk 2 above)
Comment (MishterKirby) [reply README.md]: Yeah, I kinda forgot about that.

**Review by MishterKirby — COMMENTED** [review]
How about now?

**Review by elringus — APPROVED** [review]
Approved, sorry for the delay.

**Comment by elringus** [comment]
Thanks for the corrections!

**Pull request merged**